permissions/plugin-dtkit-api.yml

```diff
@@ -1,6 +1,6 @@
 ---
 name: "dtkit-api"
-github: "jenkinsci/dtkit-libdtkit"
+github: "jenkinsci/dtkit-plugin"
 paths:
 - "org/jenkins-ci/plugins/dtkit-api"
```

Comment:
Given now IIUC you've got only one artifact, this ought to be enough?

Reply:
plugin-dtkit-api.yml refer to the https://github.com/jenkinsci/dtkit-plugin repository that contains just only one plugin artifact.